### DISCUSSION TITLE
How the user can edit his comment on any system, any browser, and any time. ???

### DISCUSSION THREAD

**Discussion by Shubhamsharma585** [posativ/isso#964] [Ideas]
My doubt is that I am already authenticating users on my platform, now I have the user's email id to identify the owner of comments. how I can avoid the cookies mechanism **so that the user can edit his comment on any system, any browser, and any time. ???**

**Comment by BBaoVanC** [comment]
I think you accidentally submitted this as a duplicate of #963

**Reply by ix5** [reply]
Removed the duplicate.
